src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -51,7 +51,7 @@ export class IssueReporter extends Disposable {
 	private hasBeenSubmitted = false;
 	private openReporter = false;
 	private loadingExtensionData = false;
-	private changeExtension = false;
+	private changeExtension: string = '';
```

Comment:
nit: this is essentially "selectedExtensionId" which may be a bit more clear since it's set by the "extension-selector"

Reply:
also nit
```suggestion
	private changeExtension = '';
```